src/rpn/rpn.test.ts

```diff
@@ -1,21 +1,33 @@
-import { rpn } from "./rpn";
+import {rpn} from "./rpn";
 
 describe("rpn()", () => {
-    it("returs number when passed", () => {
-      expect(rpn("2")).toBe(2);
+    it("returns number when passed", () => {
+        expect(rpn("2")).toBe(2);
     });
 
+    it("correctly adds 2 numbers, when 0 included", () => {
+        expect(rpn("2 0 +")).toBe(2);
+    });
 
-  it("correctly adds 2 numbers", function () {
```

Comment:
We use arrow functions everywhere, so to stick to the convention, I also rewritten this function.